### DISCUSSION TITLE
Mi Automatically Unstaked in FireFly

### DISCUSSION THREAD

**Discussion by sidonoghue** [iotaledger/firefly#2128] [Help & Support]
I used Firefly to stake my mIOTA for ASMB and SHIMMER on 30 Dec, this shows up in my recent transactions.  I just logged back into Firefly and it's no longer staked.  Is this a bug?  "Unstaking" does not show up in my recent transactions, so I'm not sure how long it's been unstaked.  I transferred a 200 Mi to my wallet on 5 Jan and left them unstaked so I could send 100Mi for the TangleLabs NFT drop and leave my staked coins untouched.  Any ideas?

**Comment by MiKeF05** [comment]
If you have sent funds from the staked wallet then your staked funds were unstaked. Otherwise the transaction can't be done. So everytime you send from your staked wallet you may need to stake again.

**Reply by sidonoghue** [reply]
Thanks Mike, If I had 200Mi unstaked and only sent 100Mi to TangleNFT, I don't understand why the staked coins were affected.  If  you transfer anything out of the wallet everything becomes unstaked without warning:  not a good user experience...

**Comment by MiKeF05** [comment]
Normally there is a warning when you send from a staked wallet. This should appear directly in the sending process.
Since IOTA uses the UTXO model and you have 200Mi unstaked tokens in the wallet that doesn't necessarily mean that the amount you send is used from the unstaked 200Mi.
